cli/scorecard/score.go

```diff
@@ -269,3 +274,17 @@ func (inventory *InventoryConfig) Score(config *ScoringConfig, outputPath string
 
 	return nil
 }
+
+func uniqueViolation(violations []*validator.Violation) []*validator.Violation {
+    uniqueViolationMap := make(map[string]*validator.Violation)
+    for _, v := range violations {
+		b, _ := json.Marshal(v)
+		hash := md5.Sum(b)
+        uniqueViolationMap[string(hash[:])] = v
+    }
+    uniqueViolation := make([]*validator.Violation, 0, len(uniqueViolationMap))
+    for _, v := range uniqueViolationMap {
+        uniqueViolation = append(uniqueViolation, v)
+    }
+    return uniqueViolation
+}
```

Comment:
nit. General practice is to have newline at end of files, might want to check your editor settings.

Reply:
done!